admin/index.js

```diff
@@ -1,11 +1,11 @@
 module.exports = AccountAdmin
 
-var getSession = require('../helpers/get-session')
+var getSession = require('../utils/get-session')
```

Comment:
Actually I always liked "helpers" more than utils :) I wouldn’t mind to settle on helpers, sounds more human somehow